src/manifest.json

```diff
@@ -15,7 +15,7 @@
   "icons": {
     "48": "icons/bell_icon.svg"
   },
-  "version": "1.0.14",
+  "version": "1.0.15",
```

Comment:
Geez, we should really fix this duplication sometime…
Maybe I'll con @ericawright into doing it when I'm out of town…  😉